app/viewmodels/helpers/events/EventsHelper.scala

```diff
@@ -81,6 +84,20 @@ class EventsHelper @Inject()(
       )
     )
 
+  private def ie810Html(event: MovementHistoryEvent, movement: GetMovementResponse)(implicit messages: Messages): Html = {
+    HtmlFormat.fill(
+      Seq(
+        p(classes = "govuk-body-l")(Html(messages(s"${timelineHelper.getEventBaseKey(event)}.p1"))),
+        p(classes = "govuk-body-l")(Html(messages(s"${timelineHelper.getEventBaseKey(event)}.reason.${movement.cancelMovement.get.reason.toString}"))),
+        if (movement.cancelMovement.get.reason == Other) p(classes = "govuk-body-l")(Html(messages("movementHistoryEvent.IE810.cancellationExplanation")))
+        else Html(""),
+        if (movement.cancelMovement.get.reason == Other) inset(Html(s"${movement.cancelMovement.get.complementaryInformation.getOrElse("")}"))
+        else Html(""),
+        printPage(linkContentKey = "movementHistoryEvent.printLink", linkTrailingMessageKey = "movementHistoryEvent.printMessage")
+      )
+    )
+  }
```

Comment:
```suggestion
  private def ie810Html(event: MovementHistoryEvent, movement: GetMovementResponse)(implicit messages: Messages): Html = {
    HtmlFormat.fill(
      Seq(
        movement.cancelMovement.map {
          cancelMovement =>
            HtmlFormat.fill(Seq(
              p(classes = "govuk-body-l")(Html(messages(s"${timelineHelper.getEventBaseKey(event)}.p1"))),
              p(classes = "govuk-body-l")(Html(messages(s"${timelineHelper.getEventBaseKey(event)}.reason.${cancelMovement.reason}"))),
              if (cancelMovement.reason == Other) {
                p(classes = "govuk-body-l")(Html(messages("movementHistoryEvent.IE810.cancellationExplanation")))
                inset(Html(s"${cancelMovement.complementaryInformation.getOrElse("")}"))
              }
              else {
                Html("")
              }
            ))
        },
        Some(printPage(linkContentKey = "movementHistoryEvent.printLink", linkTrailingMessageKey = "movementHistoryEvent.printMessage"))
      ).flatten
    )
  }
```

Just to simplify a bit:
- map inside `movement.cancelMovement` to remove `.get` all over, like `ie803Html` does
- Combine both `if/else` blocks as they're checking the same thing

Reply:
Also it's a minor thing, but please could this be put below `ie803Html` (or wherever it fits best) so they're in sequential order when scrolling through the file?